### ISSUE TITLE
Login with cache fail after server rebooting.

### ISSUE THREAD

**Issue by azizimusa** [Schmavery/facebook-chat-api#88]
I have implemented login with cache successfully by following guide from this issue :
````
https://github.com/Schmavery/facebook-chat-api/issues/46
````
My facebookBot chat is automatically executing if server rebooting. Problem is, when server is rebooting,  bot script fail to run. Below are the error shown on terminal :

````
/FacebookBot/node_modules/facebook-chat-api/index.js:269
    appState.map(function(c) {
             ^

TypeError: appState.map is not a function
    at loginHelper (/FacebookBot/node_modules/facebook-chat-api/index.js:269:14)
    at login (/FacebookBot/node_modules/facebook-chat-api/index.js:409:3)
    at Object.<anonymous> (/FacebookBot/facebookbot.js:33:1)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:467:10)
    at startup (node.js:134:18)
    at node.js:961:3
````

From which line and which file should I add code to clear the "cache.json" if got error ?

**Comment by bsansouci** [comment]
Hey @azizimusa,
Looking at the error it sounds like the appState you give isn't an array. You should check that it is. We should probably warn for that @schmavery. 
To answer your question, we don't have a nice way to tell you that you should refresh the cache. One way is to delete when you get an error back inside the login callback. That should work in most cases.

**Comment by Schmavery** [comment]
Fixed by #97
